go-tour/loops-and-functions.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"fmt"
+	"math"
+)
+
+//Sqrt returns a square root of number
+func Sqrt(x float64) float64 {
+	z := 1.0
+	for n := 0; n <= 10; n++ {
+		z = z - ((z*z - x) / (2 * z))
+	}
+	return z
+}
+
+//Implement the square root function using Newton's method
```

Comment:
please review this [instruction](https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences)